HtmlTestRunner/result.py

```diff
@@ -79,12 +81,17 @@ def __init__(self, test_result, test_method, outcome=SUCCESS,
         self.err = err
         self.stdout = test_result._stdout_data
         self.stderr = test_result._stderr_data
-
+        self.warning_message = test_result._warning_message
+        
         self.is_subtest = subTest is not None
 
-        self.test_description = self.test_result.getDescription(test_method)
+        if hasattr(test_method, '_testMethodDoc') is True:
```

Comment:
I would prefer something with a shorter line length:
```suggestion
        doc_string = getattr(test_method, '_testMethodDoc', None)
        if isinstance(doc_string, str):
            self.test_description = doc_string.strip()
        else:
            self.test_description = None
```

Reply:
That should be covering multiple lines...